ExampleNet45Project/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using System.Net.Http;
+using SendGrid;
+using SendGrid.Helpers.Mail;
+
+class Program
+{
+    private static readonly HttpClient HttpClient = new HttpClient();
+
+    static async Task Main()
+    {
+        var configuration = new ConfigurationBuilder()
+                .AddJsonFile("appsettings.json")
+                .AddJsonFile($"appsettings.{Environment.GetEnvironmentVariable("ENVIRONMENT") ?? "Production"}.json", optional: true)
+                .AddEnvironmentVariables()
+                .Build();
+        var client = new SendGridClient(HttpClient, configuration["SendGrid:ApiKey"]);
+        var from = new EmailAddress(configuration["SendGrid:From"] ?? "test@example.com", "Example User");
+        var to = new EmailAddress(configuration["SendGrid:To"] ?? "test@example.com", "Example User");
+        var msg = new SendGridMessage()
+        {
+            From = from,
+            Subject = "Sending with Twilio SendGrid is Fun",
+            MailSettings = new MailSettings
+            {
+                SandboxMode = new SandboxMode
+                {
+                    Enable = configuration.GetValue("SendGrid:SandboxMode",true)
+                }
+            }
+        };
+        msg.AddContent(MimeType.Text, "and easy to do anywhere, even with C#");
```

Comment:
Har har :)

Reply:
Can you please clearly point out the problem?